doc/memos/rdm-draft-alamos-ieee802154-radio-hal.md

```diff
@@ -39,14 +39,14 @@ and the following acronyms and definitions:
        instructions to control the radio device.
 
 # 3. Introduction
-This document is a product of the Uniform Network Stack Integration [1] and
+This document is a product of the Uniform Network Stack Integration [[1]] and
```

Comment:
Note, that this still won't make the reference show up in the References section. I think it is preferable to change it to something like the following there

```Markdown
- [Issue on Uniform Network Stack Integration](https://github.com/RIOT-OS/RIOT/issues/13771)
```

and reference it here as a normal link "`(see [issue ...](https://github.com/...)`".